user.js

```diff
@@ -270,6 +270,53 @@ user_pref("general.buildID.override",				"20100101");
 // https://github.com/pyllyukko/user.js/issues/120
 user_pref("browser.display.use_document_fonts",			0);
 
+// PREF: Prevent sites from using links like mailto to launch external applications.
```

Comment:
`// PREF: Prevent sites from using URLs such as mailto:, irc:, magnet: ... to launch external applications`